UIKitHomework/UIKitHomework/SettingOfLabelViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  SettingOfLabelViewController.swift
+//  UIKitHomework
+//
+//  Created by Polina on 05.07.2021.
+//
+
+import UIKit
+
+class Setting {
```

Comment:
```suggestion
final class Setting {
```
и по хорошему в отдельный файл бы вынести, а то потом где искать эти Setting, ну и имя бы дать TextSettings что то в таком духе, а то не понятно что за настройки